xray/audit/python/python_test.go

```diff
@@ -30,6 +55,23 @@ func TestBuildPipDependencyListSetuppy(t *testing.T) {
 	}
 }
 
+func TestPipDependencyListRequirementsFallback(t *testing.T) {
+	// Create and change directory to test workspace
+	_, cleanUp := audit.CreateTestWorkspace(t, filepath.Join("pip-project", "requirementsproject"))
+	defer cleanUp()
+	// No requirements file field specified, expect the command to use the fallback 'pip install -r requirements.txt' command
+	rootNode, err := BuildDependencyTree(pythonutils.Pip, "")
+	assert.NoError(t, err)
+	assert.Len(t, rootNode, 1)
+	if len(rootNode[0].Nodes) > 2 {
```

Comment:
if the length of rootNode[0].Nodes is 1 or 2 the test will pass.
Should we add an assertion on the length of lrootNode[0].Nodes as well?